### DISCUSSION TITLE
Don't exclude the `fuzz` tag when using `-dast / -fuzz` option

### DISCUSSION THREAD

**Discussion by zrquan** [projectdiscovery/nuclei#5508] [Ideas]
It seems that some people (#5340 and me...) think that the `-dast / -fuzz` can be used to execute templates with the fuzz tag. Maybe when the `-dast` parameter is used, the fuzz tag should not be excluded by `.nuclei-ignore`. 🤔

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
